gympass/yoga
# 🚀 Update Stepper inner text components and colors

[![JIRA Issue](https://img.shields.io/badge/issue-JIRA-blue.svg)](https://gympass.atlassian.net/browse/CIB-265)



## Description 📄



The Stepper started to use new colors from the theme instead of global colors and the new `Text.Outline` as the `Label`, matching future theme updates.

## Platforms 📲




## Type of change 🔍


## How Has This Been Tested? 🧪



- Add v3theme support at both doc and labnative environments (maintaining current default theming);
- Build normally;
- Check current and v3theme designs

- [x] Snapshot Test

## Checklist: 🔍

- [x] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [x] Layout matches design prototype: [FIGMA](https://www.figma.com/file/Hsrgmg4oLjJxYcJHaFeuJP/%F0%9F%8E%A8-Yoga-3?type=design&node-id=1310%3A2&mode=design&t=EAgW7iZ2lKgKKpzO-1)

## Screenshots 📸



| environment | screenshot |
| ------ | ----- |
| web | <img width="864" alt="image" src="https://github.com/gympass/yoga/assets/5889973/ae618341-2ec4-4163-936d-ffbd09380ac4"> |
| native | <img width="864" alt="image" src="https://github.com/gympass/yoga/assets/5889973/86d82e2f-697c-4445-9da7-c9b21c9e4e2d"> |
